bin/NpmReleaseCommand.re

```diff
@@ -623,7 +623,13 @@ let make =
                 let f = ((publicName, _innerName)) => {
                   let binName =
                     switch (System.Platform.host) {
-                    | Windows => publicName ++ ".exe"
+                    | Windows =>
+                      let publicNameLength = String.length(publicName);
+                      if (String.sub(publicName, publicNameLength - 4, 4) == ".exe") {
```

Comment:
I think this fail if length of `publicName` is less than 4. Maybe just wrap it in `Path.t` and use its functions to check/add extension.